src/ontology/uberon-edit.obo

```diff
@@ -222500,6 +222501,19 @@ intersection_of: connects UBERON:0002396 ! vomer
 relationship: dc-contributor https://orcid.org/0000-0001-8415-4848
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500110
+name: synsphenion
+def: "A craniometric point along the midline of the intersphenoid suture." [ISBN:9780873659505]
+relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500111
+name: intersphenoid suture
+def: "A suture joining the basisphenoid bone and the presphenoid bone of a sphenoid bone. This fuses at birth in humans." [ISBN:9780873659505]
+synonym: "intersphenoid synchondrosis" EXACT []
```

Comment:
Do you have a dbxref to support this this? Is it the same as the definition?

Reply:
"This fuses at birth in humans."

From https://www.ajronline.org/doi/abs/10.2214/ajr.104.1.184 (N.B. from 1968):
"The intersphenoid synchondrosis remains patent in normal children until the age of approximately 3 years."

Reply:
thank you, I will add the xrefs and will be sure to do that in the future too.